### PR TITLE
GH1595: Added Alistair and Martin names

### LINKED ISSUE #1595: Add Alistair and Martin names to all required places
In the same way that @patriksvensson @devlead and @gep13 names appear in various locations within the code base, the same should happen with @agc93 and @mholo65.

PR incoming.

### PR DESCRIPTION
**Pull request by gep13**
Fixes #1595

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Make sure you've read the [contribution guidelines](http://cakebuild.net/contrib
 
 ## License
 
-Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman and contributors.
+Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors.
```
Comment (mholo65) [inline README.md]: Remove `,` after Martin Björkström

File: nuspec/Cake.CoreCLR.nuspec [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <id>Cake.CoreCLR</id>
     <version>0.0.0</version>
-    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors</authors>
+    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors</authors>
```
Comment (mholo65) [inline nuspec/Cake.CoreCLR.nuspec]: Remove `,` after Martin Björkström

File: nuspec/Cake.Portable.nuspec [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <id>cake.portable</id>
     <title>Cake.Portable</title>
     <version>0.0.0</version>
-    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors</authors>
+    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors</authors>
```
Comment (mholo65) [inline nuspec/Cake.Portable.nuspec]: Remove `,` after Martin Björkström

File: nuspec/Cake.nuspec [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <id>Cake</id>
     <version>0.0.0</version>
-    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors</authors>
+    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors</authors>
```
Comment (mholo65) [inline nuspec/Cake.nuspec]: Remove `,` after Martin Björkström

File: nuspec/Cake.symbols.nuspec [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <id>Cake</id>
     <version>0.0.0</version>
-    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors</authors>
+    <authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors</authors>
```
Comment (mholo65) [inline nuspec/Cake.symbols.nuspec]: Remove `,` after Martin Björkström

File: src/Cake.Common.Tests/Unit/Solution/Project/Properties/AssemblyInfoParserTests.cs [hunk 6]
```diff
@@ -144,7 +144,7 @@ public void Should_Read_Configuration(string value, string expected)
             }
 
             [Theory]
-            [InlineData("Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors", "Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors")]
+            [InlineData("Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors", "Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors")]
```
Comment (mholo65) [inline src/Cake.Common.Tests/Unit/Solution/Project/Properties/AssemblyInfoParserTests.cs]: Remove `,` after Martin Björkström

File: src/Cake.Common/project.json [hunk 7]
```diff
@@ -3,7 +3,7 @@
   "description": "Provides aliases (extension methods on Cake context) that support CI, build, unit tests, zip, signing, etc. for Cake.",
   "copyright": "Copyright (c) .NET Foundation and contributors",
   "authors": [
-    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors"
+    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors"
```
Comment (mholo65) [inline src/Cake.Common/project.json]: Remove `,` after Martin Björkström

File: src/Cake.Core/project.json [hunk 8]
```diff
@@ -3,7 +3,7 @@
   "description": "The Cake core library.",
   "copyright": "Copyright (c) .NET Foundation and contributors",
   "authors": [
-    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors"
+    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors"
```
Comment (mholo65) [inline src/Cake.Core/project.json]: Remove `,` after Martin Björkström

File: src/Cake.Testing/project.json [hunk 9]
```diff
@@ -3,7 +3,7 @@
   "description": "Contains testing utilities for Cake.",
   "copyright": "Copyright (c) .NET Foundation and contributors",
   "authors": [
-    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors"
+    "Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors"
```
Comment (mholo65) [inline src/Cake.Testing/project.json]: Remove `,` after Martin Björkström

File: README.md (hunk 1 above)
Comment (gep13) [inline README.md]: @mholo65 Yeah, I am not sure on that one...

http://www.grammarbook.com/punctuation/commas.asp

Thoughts?

Comment (mholo65) [inline README.md]: @gep13 in these simple series I do find the comma unnecessary as removing it wouldn't lead to any misunderstandings. However, both works and are grammatically correct.

Before this change, there was no comma between the last name and contributors.

Does anyone else from @cake-build/cake-team have any thoughts on this?

**Pull request merged**